policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/five_children.yaml

```diff
@@ -0,0 +1,32 @@
+description: Manitoba phases the child benefit out for families with five children at this rate.
+brackets:
+  - threshold:
+      2022-01-01: 0
+      2023-01-01: 0
```

Comment:
if the values are exactly the same -> we use only the older date -> only use 2022-01-01:

Reply:
in all yaml files